internal/blockchain/ethereum/geth/gethProvider.go

```diff
@@ -0,0 +1,173 @@
+// Copyright © 2021 Kaleido, Inc.
```

Comment:
Style: for this and other files, is the convention `gethProvider` or `geth_provider` or `gethprovider`? Seems like I've mostly seen lowercase (both with and without underscores) - just want to be consistent across FireFly if we can.

Reply:
Yeah looks like `geth_provider` would be consistent with the FireFly core. I'll rename these files to be consistent.